### DISCUSSION TITLE
How would I disable clearing the placeholder?

### DISCUSSION THREAD

**Discussion by HaloElite** [soc221b/vue-next-select#403] [General]
When selecting the same option again it's clearing the selected value. How would I prevent that behavior?
Using a watcher works. But the placeholder is still getting a lighter greyish color.

```
        watch(valuestorage, (newvalue, oldvalue) => {
            if (newvalue === null) {
                valuestorage.value = oldvalue;
            }
        })
```

**Comment by soc221b** [comment]
@HaloElite 
To achieve this you could use `min` option: https://iendeavor.github.io/vue-next-select/api-reference.html#min .

**Reply by HaloElite** [reply]
I totally missed that while reading the docs. Thanks!
